integrations/integration.php

```diff
@@ -74,6 +87,11 @@ abstract class Integration {
 	public function __construct( string $slug ) {
 		$this->slug = $slug;
 
+		// Registers child integrations if any.
+		foreach ( $this->child_integrations as $integration ) {
+			IntegrationsSingleton::instance()->register( $integration );
+		}
```

Comment:
Do we expect this child logic to be reused by other integrations? If not, maybe it should live in overridden methods of `VipComposableBlocksIntegration` to keep the base class simple.